iquip/apps/scheduler.py

```diff
@@ -0,0 +1,307 @@
+"""App module for showing the experiment list."""
+
+from typing import Optional, Tuple
+
+from PyQt5.QtGui import QPainter
+from PyQt5.QtCore import Qt, QObject, QAbstractListModel, QModelIndex, QMimeData
+from PyQt5.QtWidgets import (
+    QStyleOptionViewItem, QWidget, QLayout, QLabel, QListView,
+    QPushButton, QHBoxLayout, QVBoxLayout, QAbstractItemDelegate, QAction
+)
+
+import qiwis
+from iquip.protocols import ExperimentInfo
+
+def delete_items_of_layout(layout: QLayout):
+    """Auxiliary function for deleting components in the layout.
```

Comment:
Does this actually _delete_ the components? Did you search for how we can delete the Qt objects properly? (because I always forget about that...)